Detectors/TOF/compression/src/CompressorTask.cxx

```diff
@@ -156,7 +156,7 @@ void CompressorTask<RDH, verbose, paranoid>::run(ProcessingContext& pc)
     }
 
     v.resize(headerOut.payloadSize);
-    pc.outputs().adoptContainer(output, std::move(v), true, header::gSerializationMethodNone);
```

Comment:
Yes, indeed the `true` should only be used for the CCDB object cases, were we keep them in memory for more than one timeframe.

Reply:
Thanks for the check, then this explains what I observed